doc/manuals/admin/upgrading_crossing_2-2-0.md

```diff
@@ -0,0 +1,92 @@
+
+-   Stop contextBroker
+-   Remove previous contextBroker version
+
+        yum remove contextBroker
+
+-   [Take a backup of your DBs](database_admin.md#backup) (this is just a safety measure in case any problem occurs,
+    e.g some script gets interrupted before finished and your database data ends in an incoherent state)
+-   Download the following script:
+    -   [check_metadata_id.py](https://github.com/telefonicaid/fiware-orion/blob/2.2.0/scripts/managedb/check_metadata_id.py)
+-   Install pymongo (it is a script dependency) in case you don't have it previously installed
+
+        pip-python install pymongo
+
+-   Apply the `check_metadata_id.py` script to your DBs, using the following command (where 'db' is the database name). Note that
+    if you are using [multitenant/multiservice](database_admin.md#multiservicemultitenant-database-separation) you need to apply the
+    procedure to each per-tenant/service database. The script can take a while, so an interactive progress counter is shown.
+
+        python check_metadata_id.py orion
+
+-   If you get some message like this one, then check the "Dealing with metadata ID" section.
+
+        entity {u'type': u'...', u'id': u'...', u'servicePath': u'...'}: found attr <...>, metadata id <...>, value <...>
+
+-   If you don't get the above message, then you data doesn't have any ID and you can proceed with next step-
+
+-   If you get any of the following messages, there is some problem that needs to be solved before going to the next step. Check the
+    "Troubleshooting" section below.
+
+        WARNING: some problem was found during the process.
+        ERROR: document ... change attempt failed
+
+-   Install new contextBroker version. (Sometimes the commands fails due to yum cache. In that case, run "yum clean all" and try again)
+
+        yum install contextBroker
+
+-   Start contextBroker
+
+Note that the rpm command demands superuser privileges, so you have to run it as root or using the sudo command.
+
+## Dealing with metadata ID
+
+In the case you have metadata ID in your database, you need to remove them. There are two ways of doing so:
+
+-   Using the Orion API with the old version of Orion you were using (recommended). You can use the
+    `DELETE /v1/contextEntities/{entityId}/attributes/{attrName}/{attrId}` or
+    `DELETE /v1/contextEntities/type/{entityType}/id/{entityId}/attributes/{attrName}/{attrId}` operations.
+
+-   Using the script `check_metadata_id.py` in autofix mode. In order to do so, edit the script and change the line `autofix = None`
+    to either `autofix = 'as_new_attrs'` or `autofix = 'as_metadata'`.
+    - In `as_new_attrs` mode each attribute with ID is transformed to a new attribute in the form `<attrName>:<id>` (e.g. if you
+      have an attribute `temperature` with metadata ID `id1` then it will be transformed to `temperature:id1`. In the rare case the
```

Comment:
- Add a close bracket
transformed to `temperature:id1`**)**.

Reply:
Fixed in e97e48f1c4ef7b7f3b2bf995eed923df1cbc4ee9. Thanks!